packages/plugin-aws-elasticsearch/src/plugin.ts

```diff
@@ -0,0 +1,129 @@
+import assert = require('assert')
+import { resolve } from 'path'
+import retry = require('bluebird-retry')
+import Bluebird = require('bluebird')
+import Errors = require('common-errors')
+import Elasticsearch = require('@elastic/elasticsearch')
+import { NotFoundError } from 'common-errors'
+import { PluginTypes, Microfleet } from '@microfleet/core'
+import * as AWS from 'aws-sdk'
+import createAwsElasticsearchConnector from './utils/createAwsElasticsearchConnector'
```

Comment:
using default imports is not recommended, named imports are better for consistency of the code